presentations/basic-types/slides.adoc

```diff
@@ -37,13 +37,31 @@ Overflows trigger a trap in Debug mode, but not in release mode. This behaviour
 
 == Floats
 
-Rust also comes with floats of all standard sizes: f32, f64
+Rust also comes with floats of all standard sizes: `f32`, `f64`
```

Comment:
This looks like a good place to note that the default float type is `f64`.

(Of course this does not need to be done in this PR)